wsi/drm/drmbuffer.h

```diff
@@ -72,6 +72,10 @@ class DrmBuffer : public OverlayBuffer {
     return offsets_;
   }
 
+  uint32_t GetTilingMode() const override {
+    return tiling_mode_;
```

Comment:
We had to revert https://github.com/intel/minigbm/commit/7b767e105fa63154c19cb92f16f3b447ebb485ee as it broke media. We will need to re-enable this but we can do while getting this committed.

Reply:
you means that we can continue this commit, then after it merge, you will also recommit  the minigbm patch also, right?